cypari2/gen.pyx

```diff
@@ -38,6 +38,9 @@ AUTHORS:
 
```

Comment:
Given that you rely on *explicit* conversions in many other places, I think it's better to be explicit everywhere. This means dropping `cython: c_string_encoding=default` and adding `to_string` or `to_bytes` where needed.